fvm/meter/weighted/meter.go

```diff
@@ -36,12 +36,10 @@ var (
 		common.MemoryKindCharacter:               24,
 		common.MemoryKindMetaType:                0,
 		common.MemoryKindNumber:                  8,
-		common.MemoryKindArrayBase:               57,
-		common.MemoryKindArrayLength:             1024,
-		common.MemoryKindDictionaryBase:          33,
-		common.MemoryKindDictionarySize:          1024,
-		common.MemoryKindCompositeBase:           233,
-		common.MemoryKindCompositeSize:           1024,
+		common.MemoryKindArrayBase:               57 + 48,
+		common.MemoryKindDictionaryBase:          33 + 96,
+		common.MemoryKindCompositeBase:           233 + 96,
```

Comment:
It seems like the amounts are "split" into two parts for some reason – if possible, add a comment explaining why

Reply:
Thank you for adding the comment 👌